src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoRoot)\build\CodeAnalysis.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;net461;netstandard2.0;net5.0</TargetFrameworks>
```

Comment:
curious -is there a need to explicitly add net5.0?

Reply:
This is a new project so I figured it would be a good chance to use some of the new tooling. You saw CodeAnalysis.props I'm sure. That file is turning on Microsoft.CodeAnalysis.NetAnalyzers (the successor to Microsoft.CodeAnalysis.FxCopAnalyzers) & [nullable reference types](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references). The one bummer with nullable enabled is that VS only properly does analysis for netcoreapp3.1, netstandard2.1, and/or net5.0. If you turn it on for anything older, you get a lot of noise. So I added net5.0 to get proper analysis, it isn't used by any of the code directly. I know that sucks. I just moved it behind a `Debug` condition, that way it won't be part of the release. Let me know what you think about that.